src/client/table_client.rs

```diff
@@ -1169,19 +1195,19 @@ impl ObTableClientInner {
             < duration_to_millis(&self.config.metadata_refresh_interval)
     }
 
-    fn sync_refresh_metadata(&self) -> Result<()> {
+    fn sync_refresh_metadata(&self, refresh_immediately: bool) -> Result<()> {
```

Comment:
To ensure we could refresh metadata from the server every time, I leave a backdoor here for convenience to block "check interval". After tests, I found that we could lower the interval instead of skipping the interval. Now this problem is fixed.